src/doc/html/static/rustdoc.css

```diff
@@ -568,6 +568,10 @@ h4 > code, h3 > code, .invisible > code {
 	flex-basis: 100%;
 }
 
+.type-annotation {
+	font-weight: 200;
+}
+
```

Comment:
I was wondering if we should add an independent CSS file for customizations, so `rustdoc.css` can be easily updated from upstream.

Reply:
I think this is a good idea!

Reply:
Done.